ctru-rs/src/gfx.rs

```diff
@@ -138,6 +123,44 @@ impl TopScreen {
     pub fn get_wide_mode(&self) -> bool {
         unsafe { ctru_sys::gfxIsWide() }
     }
+
+    /// Returns a tuple containing a pointer to the top screen's framebuffer
+    /// (as determined by the `side`), the width of the framebuffer in pixels,
+    /// and the height of the framebuffer in pixels.
+    ///
+    /// Note that the pointer returned by this function can change after each call to this function
+    /// if double buffering is enabled
+    pub fn get_raw_framebuffer(&mut self, side: Side) -> (*mut u8, u16, u16) {
+        let mut width: u16 = 0;
```

Comment:
nit: Just a personal preference, but you don't need any of the local variable types in these functions. They get inferred from the return type.